image_view/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8)
 project(image_view)
 
+set(CMAKE_CXX_FLAGS "-std=c++0x ${CMAKE_CXX_FLAGS}")
```

Comment:
we can't ask for a package to depend on C++11 during a release cycle. Please remove this line and have your code use proper C++03 code. Thx